### PR TITLE
Cymetric 1.5.5 LINUX ONLY

### PR DESCRIPTION
**Pull request by stompsjo**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

This update is for the new micro-release of Cyclus 1.5.5. We are only implementing for Linux since the OSX conda-forge for Cyclus is not working right now.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,12 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   noarch: python
   script: python setup.py install --prefix ${PREFIX}
```
Comment (bam241) [inline recipe/meta.yaml]: this has to change. 
`script` section is probably not required.

look at #9 I issued to get inspiration.

**Review by bam241 — COMMENTED** [review]
`build` section has to be reshaped
`skip` section has to be added. (matching cycamore/cyclus ones)

will probably required `setuptool` or something similar in `requirement` `build` section

requirement on cyclus version should be something like `{{ version }}`

**Comment by bam241** [comment]
also this is missing a `build.sh` script.

look at #9 for inspiration

**Comment by bam241** [comment]
make sure there is a `run_test.sh` script. to check files have gin properly installed and nosetest ran correctly. 
(look at Cyclus-feedstock for example)

File: recipe/meta.yaml (hunk 1 above)
Comment (stompsjo) [reply recipe/meta.yaml]: I made all of these changes!

**Comment by stompsjo** [comment]
@conda-forge-admin please rerender

**Comment by bam241** [comment]
@stompsjo, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,29 +11,35 @@ source:
...
 requirements:
   build:
+    - setuptools
+  host:
     - python
   run:
     - python
-    - cyclus >=1.5.*
+    - cyclus {{ version }}
     #- pyne  # this is working locally, but not on conda-forge for some weird reason
```
Comment (bam241) [inline recipe/meta.yaml]: maybe remove that also as it is not required

Comment (stompsjo) [reply recipe/meta.yaml]: This pyne line has been removed.

**Comment by stompsjo** [comment]
@bam241 This looks to be working!

**Comment by bam241** [comment]
LGTM!
Thx @stompsjo

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,29 +11,35 @@ source:
...
 build:
-  number: 1
-  noarch: python
-  script: python setup.py install --prefix ${PREFIX}
+  number: 0
+  skip: True  # [win or osx or py<=27]
 
 requirements:
   build:
+    - setuptools
```
Comment (bam241) [inline recipe/meta.yaml]: @stompsjo  Could you check if this is actually required?
I am having doubt...

Comment (stompsjo) [reply recipe/meta.yaml]: @bam241 I removed it, will wait to see if the builds pass.

**Comment by bam241** [comment]
Thx @stompsjo !
You rock!

**Pull request merged**